community/postsrsd/postsrsd.initd

```diff
@@ -0,0 +1,29 @@
+#!/sbin/openrc-run
+
+description="Postfix SRS Daemon"
+pidfile="/run/postsrsd.pid"
+command="/usr/sbin/postsrsd"
+command_args="-D -d ${SRS_DOMAIN} -s ${SRS_SECRET} -f ${SRS_FORWARD_PORT}
+              -r ${SRS_REVERSE_PORT} -p \"${pidfile}\" -u ${RUN_AS}
+              -c \"${CHROOT}\" -a \"${SRS_SEPARATOR}\""
```

Comment:
@jirutka Those defaults (names) are coming from the [project](https://github.com/roehling/postsrsd/blob/master/init/postsrsd.default.in). Most of the defaults are defined in there too. The only thing you are required to set yourself is the SRS domain.

Do you think it makes sense to copy these defaults over in the initd file?